src/test/java/org/springframework/data/couchbase/core/mapping/MappingCouchbaseConverterTests.java

```diff
@@ -33,7 +33,9 @@
 import java.util.TreeMap;
 import java.util.TreeSet;
 
-import org.joda.time.LocalDateTime;
+import java.time.LocalDateTime;
```

Comment:
are we still supporting pre java 8? or are we on java 8 now completely for spring data